### PR TITLE
update page title to prepend 'Workshop:'

### PR DESCRIPTION
**Pull request by shughes26**
Addressing [this comment](https://github.com/pulumi/pulumi-hugo/pull/1734#discussion_r920469981)

### CODE REVIEW

**Comment by shughes26** [comment]
New title is longer than the maximum number of characters (65 characters, limit 60). Will leave this here until we've got a better idea of what to change the title to.

File: themes/default/content/resources/introduction-to-infrastructure-as-code-with-yaml/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 # Name of the webinar.
-title: "Introduction to Infrastructure as Code with Pulumi YAML"
+title: "Workshop: Introduction to Infrastructure as Code with Pulumi YAML"
```
Comment (zchase) [inline themes/default/content/resources/introduction-to-infrastructure-as-code-with-yaml/index.md:3]:
```suggestion
title: "Workshop: Introduction to Infrastructure as Code with YAML"
```

I'd just drop the "Pulumi" before "YAML".

**Comment by susanev** [comment]
@shughes26 @zchase are yall still wanting to get this in?

**Comment by susanev** [comment]
gonna go ahead and close this out, feel free to reopen if yall still would like to get this in

**Pull request closed without merging**